lib/code_corps_web/controllers/user_controller.ex

```diff
@@ -68,9 +71,9 @@ defmodule CodeCorpsWeb.UserController do
     conn |> json(hash)
   end
 
-  defp login(conn, _opts) do
-    Plug.Conn.register_before_send(conn, &do_login(&1))
-  end
+  # defp login(conn, _opts) do
```

Comment:
@begedin any idea why commenting these lines (and the plug :login) above out allows for a passing test suite?  Otherwise I get this [line](https://github.com/code-corps/code-corps-api/blob/develop/test/lib/code_corps/github/installation_test.exs#L46) and this [line](https://github.com/code-corps/code-corps-api/blob/develop/test/lib/code_corps/github/installation_test.exs#L75) failing

Reply:
I'm not sure, but I would expect the plug to cause things to fail either way. It was added mainly to get around some ja_resource quirks.

What I would do is remove `plug :login when action in [:create]`, as well as `defp login` and `defp do_login` completely.

Then, in `create/2`, I would add a `|> Guardian.Plug.sign_in(user)` between `conn` and `|> put_status(:created)`, or really, anywhere in that chain.

That would be the most explicit and easiest to read approach for dealing with it anyway.

Reply:
@begedin is this b/c the user is signed in automatically after creating a user?